addons/mail/static/src/models/channel_member_list_view.js

```diff
@@ -1,66 +1,73 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
-import { one } from '@mail/model/model_field';
-import { clear, insertAndReplace, replace } from '@mail/model/model_field_command';
+import { attr, one } from '@mail/model/model_field';
+import { clear, replace } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'ChannelMemberListView',
     identifyingFields: [['chatWindowOwner', 'threadViewOwner']],
+    lifecycleHooks: {
+        _created() {
+            this.channel.fetchChannelMembers();
+        },
+    },
     recordMethods: {
+        /**
+         * Handles click on the "load more members" button.
+         */
+        async onClickLoadMoreMembers() {
+            this.channel.fetchChannelMembers();
+        },
         /**
          * @private
-         * @returns {FieldCommand}
+         * @returns {Boolean}
          */
-        _computeChannel() {
-            if (this.chatWindowOwner) {
-                return replace(this.chatWindowOwner.thread);
+        _computeAreAllMemberLoaded() {
+            if (!this.channel) {
+                return;
             }
-            if (this.threadViewOwner) {
-                return replace(this.threadViewOwner.thread);
-            }
-            return clear();
+            return this.channel.memberCount < this.channel.channelMembers.length;
```

Comment:
I feel like this computes the opposite of what its name suggests. Shouldn't the condition be something like `this.channel.memberCount === this.channel.channelMembers.length`?